core/capabilities/ccip/ccipevm/msghasher_test.go

```diff
@@ -24,22 +24,25 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/smartcontractkit/chainlink-ccip/chains/solana/gobindings/fee_quoter"
-
 	"github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/ccipsolana"
-	ccipcommon "github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/common"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/logger"
 
 	"github.com/smartcontractkit/chainlink-evm/gethwrappers/ccip/generated/v1_6_0/message_hasher"
 	"github.com/smartcontractkit/chainlink-integrations/evm/assets"
 	evmtestutils "github.com/smartcontractkit/chainlink-integrations/evm/testutils"
 	"github.com/smartcontractkit/chainlink-integrations/evm/utils"
+	ccipcommon "github.com/smartcontractkit/chainlink/v2/core/capabilities/ccip/common"
 	"github.com/smartcontractkit/chainlink/v2/core/internal/testutils"
 
 	cciptypes "github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
 )
 
-var ExtraDataCodec = ccipcommon.NewExtraDataCodec(ccipcommon.NewExtraDataCodecParams(ExtraDataDecoder{}, ccipsolana.ExtraDataDecoder{}))
+var extraDataCodec = ccipcommon.NewExtraDataCodec(
+	ccipcommon.ExtraDataCodecParams{
+		EVMExtraDataDecoder:    ExtraDataDecoder{},
+		SolanaExtraDataDecoder: ccipsolana.ExtraDataDecoder{},
+	})
```

Comment:
A circular dependency in tests prevent using the default instantiations in `ccipevm` and `ccipsolana`. I tried to switch some tests to use a `ccipevm_test` package, but it was slightly too involved for this PR due to a private helper function being used.

Reply:
I have a refactor PR for this https://github.com/smartcontractkit/chainlink/pull/16363

Reply:
It looks like that PR has been open for a couple months. Should I merge in this small change or wait for yours?

Reply:
It was on hold because of Blaž e2e PR. Now it's merged, I reopened it. Can we get https://github.com/smartcontractkit/chainlink/pull/16363 merged first ? I can support this default config in a following PR or you can resolve conflicts here after